virtualization/windowscontainers/manage-containers/container-networking.md

```diff
@@ -10,344 +10,248 @@ ms.service: windows-containers
+## Specify the Name of a Network to the HNS Service
 
-> Note: VFP is not available on Windows 10 Anniversary Edition (Version 1607). 
+> Applies to all network drivers 
 
-```none
-Get-Service vfpext
-```
+Ordinarily, when you create a container network using `docker network create`, the network name that you provide is used by the Docker service but not by the HNS service. If you are creating a network, you can specify the name that it is given by the HNS service using the option, `-o com.docker.network.windowsshim.networkname=<network name>` to the `docker network create` command. For instance, you might use the following command to create a transparent network with a name that is specified to the HNS service:
 
 ```none
-C:\> docker network create -d l2bridge --subnet=192.168.1.0/24 --gateway=192.168.1.1 MyBridgeNetwork
+C:\> docker network create -d transparent -o com.docker.network.windowsshim.networkname=MyTransparentNetwork MyTransparentNetwork
 ```
 
-Only static IP assignment is supported with l2bridge networks.
-
-> When using an l2bridge network on an SDN fabric, only dynamic IP assignment is supported. Reference the [Attaching Containers to a Virtual Network](https://technet.microsoft.com/en-us/windows-server-docs/networking/sdn/manage/connect-container-endpoints-to-a-tenant-virtual-network) topic for more information.
-
-## Other Operations and Configurations
+## Bind a Network to a Specific Network Interface
 
-> We are constantly working on improving Docker on Windows. To make sure you have access to all of the latest functionality, confirm that you are using the latest version of the Docker Engine. You can check your docker version using `docker -v`. Reference the [Docker Engine on Windows](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/configure-docker-daemon) topic for guidance on configuring Docker.
+> Applies to all network drivers except 'nat'  
 
-### List Available Networks
+To bind a network (attached through the Hyper-V virtual switch) to a specific network interface, use the option, `-o com.docker.network.windowsshim.interface=<Interface>` to the `docker network create` command. For instance, you might use the following command to create a transparent network which is attached to the "Ethernet 2" network interface:
 
 ```none
-# list container networks
-C:\> docker network ls
-
-NETWORK ID          NAME                DRIVER              SCOPE
-0a297065f06a        nat                 nat                 local
-d42516aa0250        none                null                local
+C:\> docker network create -d transparent -o com.docker.network.windowsshim.interface="Ethernet 2" TransparentNet2
 ```
 
-### Remove a Network
-
-Use `docker network rm` to delete a container network.
+> Note: The value for *com.docker.network.windowsshim.interface* is the network adapter's *Name*, which can be found with:
 
-```none
-C:\> docker network rm <network name>
+>```none
```

Comment:
From what I can see in the actual Github markdown viewer, you can't combine `>` with the triple-backtick blocks. The output is all messed up to the VFP section.